client_interface.py

```diff
@@ -30,16 +30,19 @@ def on_text(self, text):
         Wait for user input on keyboard and react for it.
         """
         handle_text(self.state, text)
-        self.send_to_server(self.state)
+        if self.state.selection_confirmed:
```

Comment:
stejně tak tady mi to není jasné. ` if self.state.selection_confirmed:` to jako říká - pokud je ta hodnota True? Já měla za to, že pokud selection_confirmed má nějakou hodnotu, prostě pro moje chápání bych napsala ` if self.state.selection_confirmed == True`  tak mi jen potvrď tento zkrácený zápis že to znamená že to je True :-D

Reply:
`if` vyhodnotí logickou hodnotu výrazu. Pokud je podmínka splněná (vrátí `True`), provede se kus kodu pod ní. Pokud není splněná (vrátí `False`), blok kódu pod ní se neprovede. Proto porovnávat `== True` nemá smysl, podmínka bude stejně vyhodnocená jako T/F. Plus bacha, bavily jsme se někdy před nedávnem, že při vyhodnocení výrazu, např. v testech, se nepoužívá `==` u bool hodnot, ale `is`, na to narazíš ve většíně materiálů na netu.

Reply:
Děkuji za vysvětlení :)